### PR TITLE
#582 - Restrict non BC Public Institutions from accessing Routes for BC Public institutions

### PR DESCRIPTION
**Pull request by dheepak-aot**
Based on the route meta property `allowOnlyBCPublic` block access for non BC Public Institution users from accessing routes which are for BCPublic institutions.

### CODE REVIEW

File: sources/packages/web/src/router/InstitutionRoutes.ts [hunk 1]
```diff
@@ -468,6 +469,7 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
             component: InstitutionStudentProfile,
             meta: {
               clientType: ClientIdType.Institution,
+              allowOnlyBCPublic: true,
```
Comment (ann-aot) [inline sources/packages/web/src/router/InstitutionRoutes.ts:472]: adding `allowOnlyBCPublic` to the parent (line 457) will it protect all children? As discussed, you can just try and check

Comment (dheepak-aot) [reply sources/packages/web/src/router/InstitutionRoutes.ts]: I tested by using allowOnlyBCPublic in parent onluy and it works perfectly. thanks.

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @dheepak-aot

**Review by andrepestana-aot — APPROVED** [review]
LGTM

File: sources/packages/web/src/router/InstitutionRoutes.ts (hunk 1 above)
Comment (ann-aot) [reply sources/packages/web/src/router/InstitutionRoutes.ts]: Thanks for trying it. As discussed, we can test and remove the other meta from the children in another ticket

**Review by ann-aot — APPROVED** [review]
Good work @dheepak-aot 👍

**Review by andrewsignori-aot — APPROVED** [review]
Nice work, looks good 👍

**Pull request merged**